dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -185,11 +193,10 @@ class BuildTest {
 }
 
 class MemoryTest {
-  MemoryTest(this.testDirectory, this.testTarget, this.timelineFileName, this.packageName, { this.os });
+  MemoryTest(this.testDirectory, this.testTarget, this.packageName, { this.os });
```

Comment:
Let's document what `testTarget` parameter means, and what happens when it's not specified.